### DISCUSSION TITLE
Bypass Rate Limit and Concurrent PR limit for certain dependencies

### DISCUSSION THREAD

**Discussion by mbmx10** [renovatebot/renovate#27554] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

irrelevant for the question

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### What is your question?

We use a rate limit to save load on the build server.

Is there a way to bypass the hourly rate limit and prConcurrentLimit for certain dependencies?

### Logs (if relevant)

not relevant

**Comment by rarkins** [comment]
Not supported, see #10546

**Reply by mbmx10** [reply]
Thanks, I'll close this discussion.
